tests/Settings/config_parser_files/weird_values.cfg

```
@@ -6,6 +6,11 @@ even_more_weirdness = [yay!]
 woah_weird = =(200 if the_value != "whatever" or more_weirdness == "derp" else 0)
 lesser_equal = =60 if 1 <= 2 else 30
 greater_equal = =20 if 1 >= 2 else "yay"
+key_value_in_string = G1
+	; speed_z = {speed_z}
```

Use spaces instead of tab.

```suggestion
    ; speed_z = {speed_z}
```